.github/workflows/handle_stalled.yml

```
@@ -1,4 +1,4 @@
-name: "Handled Stalled Issues & PRs"
+name: "Handled Stalled Issues & Merge Requests"
```

Missed changing "Merge Requests" here to "Pull Requests", also not sure if you meant "Stalled" or "Stale". I assume "Handle" is more appropriate than "Handled" for the job name too?

```suggestion
name: "Handle Stale Issues & Pull Requests"
```